service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/ControlledAiNotebookInstanceResource.java

```diff
@@ -159,43 +159,6 @@ public Optional<UniquenessCheckAttributes> getUniquenessCheckAttributes() {
             .addParameter("instanceId", getInstanceId()));
   }
 
-  /**
```

Comment:
is this related?

Reply:
Since we do a very similar cloud sync I assumed we would have the same DRS slowness